crates/client/db/src/lib.rs

```diff
@@ -395,6 +396,7 @@ impl MadaraBackend {
             last_flush_time: Default::default(),
             chain_config,
             db_metrics: DbMetrics::register(&MetricsRegistry::dummy()).unwrap(),
+            sender_block_info: tokio::sync::broadcast::channel(1024).0,
```

Comment:
I think we can reduce the buffer to only a few blocks

Reply:
Fixed..